docs/en/operation/Metrics-System.md

```diff
@@ -126,3 +126,87 @@ to the under store.
 
 Tags are additional pieces of metadata for the metric such as user name or under storage location.
 Tags can be used to further filter or aggregate on various characteristics.
+
+## Setting Grafana for Alluxio
+
+Grafana is a metics analytics and visualization software commonly used for visualizing time series
+data. You can use Grafana to better visualize the various metrics that Alluxio collects. The software
+allows users to more easily see what's going on behind the scenes in Alluxio.
+
+Although before setting up Grafana a monitoring tool must be set up that Grafana will pull metrics from
+such as `Prometheus` or `Graphite`.
+
+### Setting up Prometheus
+
+For installing Prometheus you can refer to the Docs [here](https://prometheus.io/docs/prometheus/latest/installation/).
+
+After installing Prometheus must be configured to get metrics from Alluxio. First open the
+prometheus.yml file that the Prometheus Docker calls when starting. Once inside the .yml file multiple
+jobs have to be created for the Alluxio master and worker shown below:
```

Comment:
Instead of saying "once inside the `.yml` file" I would just write "Put the following sections under the `scrape_configs:` section of the Prometheus configuration